bdk_core/src/descriptor_tracker.rs

```diff
@@ -830,27 +897,39 @@ impl DescriptorTracker {
                 hasher.input(&txid);
             }
             let txid_hash = sha256::Hash::from_engine(hasher);
-            txid_hash == tip_txids.accum_digest
+            self.checkpoints
+                .get(&tip.height)
+                .expect("Must exist, as the reference to it exists")
+                .get_accum_digest()
+                == txid_hash
         } else {
             true
         }
     }
 }
 
+/// An Update that is obtained after a sync and is ready to
+/// be applied in the DB.
+/// Setting each field of the Update performs different update mechanisms.
```

Comment:
This comment is not correct. A very basic helpful comment would be:

> An `Update` contains a list of new transactions added from `base_tip` to `new_tip`. I don't think it's worth going further since this is going to change.